sis-local/streamlit_in_snowflake/sis.py

```diff
@@ -0,0 +1,72 @@
+from typing import Callable
+
+from streamlit import *  # noqa: F403
+from streamlit import (
+    audio,
+    bokeh_chart,
+    cache,
+    camera_input,
+    download_button,
+    experimental_get_query_params,
+    experimental_set_query_params,
+    experimental_user,
+    file_uploader,
+    image,
+    video,
+    warning,
+)
+from streamlit import write as original_write
+
+SIS_FAQ = "https://docs.google.com/document/d/1XY0wPp5tJQHivkzkcxYKjwKiW8-rreHsUwdYFmWEuwg/edit"
+
+# TODO: st.markdown should also warn when using unsafe html
+# TODO: warn for custom components
```

Comment:
```suggestion
# TODO: st.markdown should also warn when using unsafe html
# TODO: warn for custom components
# TODO: warn for components API
# TODO: st.info for cache_data and cache_resource? Since they won't crash but just won't work
```